src/Model/FetchSongsManager.php

```diff
@@ -30,22 +30,22 @@ public function showImgDir()
      */
     public function songSearch(): array
     {
-        $query = 'SELECT songs.ID_song , songs.title , songs.artist , genre.genre, songs.image_url, songs.song_url 
-                    FROM songs
-                    join genre on songs.genre_ID_genre = genre.ID_genre
-                    where songs.title like "%'.$_GET['search'].'%" or songs.artist like "%'.$_GET['search'].'%" or genre.genre like "%'.$_GET['search'].'%"';
+        $query = 'SELECT songs.ID_song , songs.title , songs.artist , genre.genre, songs.image_url, songs.song_url
+        FROM songs join genre on songs.genre_ID_genre = genre.ID_genre
+        where songs.title like "%' . $_POST['search'] . '%" or songs.artist like "%' .
```

Comment:
this is old code
$_POST needs to be replaced with $_GET

Reply:
Done